src/main/kotlin/no/nav/syfo/cronjob/dialogmotesvar/Dialogmotesvar.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.cronjob.dialogmotesvar
+
+import no.nav.syfo.dialogmote.domain.DialogmoteSvarType
+import no.nav.syfo.domain.PersonIdent
+import java.time.OffsetDateTime
+import java.util.*
+
+data class Dialogmotesvar(
+    val moteuuid: UUID,
+    val dbRef: UUID,
```

Comment:
Hva er egentlig `dbRef`?

Reply:
UUID til varsel (for arbeidsgiver/arbeidstaker) eller svar (behandler) som man skal oppdatere etter publisering (se https://github.com/navikt/isdialogmote/pull/511/files#diff-089c000586d913bb45f2394105c05c1bc978381c314a39db5094e863317260ebR40)